bin/pygrb/pycbc_pygrb_plot_snr_timeseries

```diff
@@ -133,34 +132,35 @@ ifos, vetoes = ppu.extract_ifos_and_vetoes(trig_file, opts.veto_files,
 trig_data = load_data(trig_file, vetoes)
 
 # Load (or initialize) injection data
-inj_data = load_data(found_file, vetoes, injections=True)
+inj_data = load_data(inj_file, vetoes, injections=True)
 
 # Specify HDF file keys for x quantity (time) and y quantity (SNR)
 if snr_type == 'single':
-    x_key = opts.ifo + '/' + 'end_time'
-    y_key = opts.ifo + '/' + 'snr_' + opts.ifo.lower()
+    x_key = opts.ifo + '/end_time'
+    y_key = opts.ifo + '/snr_' + opts.ifo.lower()
+    if opts.ifo.lower() != 'h1':
```

Comment:
See before. Also, this code shows up a few times, maybe a function would be useful?

Reply:
Commented this too.